api-guide.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Webhooks API guide
+sidebar_label: API guide
+sidebar_position: 30
+description: Find technical details about integrating with the Webhooks API.
+pagination_prev: Null
+pagination_next: Null
+---
+
+# API guide
+
+Webhooks alert you to your API request updates (e.g., payment *Authorized*, *Cancelled*, *Expired*, and
```

Comment:
Sorry, that term confuse me, 'API request updates' what does that mean?
What about 

'
You can use the webhooks API to get an http notification 
whenever a subscribed event happen, for example an captured payment on the ePayment platform.
'

Reply:
That sounds better. :-)